openrtb_ext/imp_liftoff.go

```diff
@@ -0,0 +1,7 @@
+package openrtb_ext
+
+type ImpExtLiftoff struct {
+	BidToken       string `json:"bid_token,omitempty"`
+	PubAppStoreID  string `json:"app_store_id,omitempty"`
+	PlacementRefID string `json:"placement_reference_id,omitempty"`
```

Comment:
Please remove the `omitempty` tag as all the parameters are defined as required in the `bidder-params/liftoff.json`

Reply:
@Sonali-More-Xandr DONE